File: Core/Entities/MedicineProduct.cs
namespace Core.Entities
{
    public class MedicineProduct : BaseEntity
    {
        public string ProductCode { get; set; }
        public string ProductName { get; set; }
        public string PackSize { get; set; }
        public string Status { get; set; }
        public double UnitTp { get; set; }
        public double UnitVat { get; set; }
        public string SorgaCode { get; set; }
       
    }
}

File: client/src/app/shared/models/medicineProduct.ts

export interface IMedicineProduct {
    id: number;
    productName: string;
    productCode: string;
    status: string; 
    unitTp: number; 
    unitVat: number; 
    sorgaCode: Date;
}
 
export class MedicineProduct implements IMedicineProduct {
    id: number=0;
    productName: string;
    productCode: string;
    status: string="Active"; 
    unitTp: number; 
    unitVat: number; 
    sorgaCode: Date;
}